spec/controllers/v0/users_controller_spec.rb

```diff
@@ -149,4 +149,26 @@
       end
     end
   end
+
+  describe '#credential_emails' do
+    let(:user) { create(:user, :loa1) }
```

Comment:
This mostly looks good, I'm going to recommend a change so that we can prove this will pull emails from multiple credentials in case a user has several associated with them:
- Create the user as `loa3` -> `let(:user) { create(:user, :loa3) }`
- Create one more `user_verification` (the current one is associated with `idme_uuid`, you could use `logingov_uuid` for the other `user_verification`)
- Confirm there are multiple emails in the response

Reply:
Oh good idea! I should've thought of that oops